web/src/components/Comments/Comments.connector.ts

```diff
@@ -40,14 +46,11 @@ function mapDispatchToProps(dispatch, ownProps) {
       )
       return dispatch(updateOutcomeComment(cellIdString, outcomeComment))
     },
-    deleteOutcomeComment: async (payload) => {
+    deleteOutcomeComment: async (headerHash: HeaderHashB64) => {
       const appWebsocket = await getAppWs()
       const projectsZomeApi = new ProjectsZomeApi(appWebsocket)
-      const outcomeComment = await projectsZomeApi.outcomeComment.delete(
-        cellId,
-        payload
-      )
-      return dispatch(deleteOutcomeComment(cellIdString, outcomeComment))
+      await projectsZomeApi.outcomeComment.delete(cellId, headerHash)
```

Comment:
bug fix here... we need to continue to pass the headerHash of the OutcomeComment through to the action, even though the return type of the async call is `void`. this was a bit of a gotcha in terms of refactoring.